src/app/components/pmapi/pmapi.service.js

```diff
@@ -228,10 +250,26 @@
 
             return deferred.promise;
         }
-
+      
+        function getMetricsMetadata(context) {
```

Comment:
I am closing this pull request, as I am already storing the data in Custom Widget Feature, I will fix it there.